### PR TITLE
UX: do not show reactions by default

### PR DESCRIPTION
**Pull request by techAPJ**

### CODE REVIEW

File: assets/stylesheets/common/question-answer.scss [hunk 1]
```diff
@@ -15,6 +15,11 @@
       border-top: 1px solid var(--primary-low);
     }
   }
+
+  // disable reactions by default
+  .discourse-reactions-actions {
```
Comment (tgxworld) [inline assets/stylesheets/common/question-answer.scss:20]: Is there a way we can do this without coupling this plugin to another plugin that has nothing to do with this plugin?

Comment (techAPJ) [reply assets/stylesheets/common/question-answer.scss:20]: I tried looking for other ways (initializers, etc) but couldn't find any. This seems to be the simplest route and can be reverted (if site admin wants to enable reactions on Q&A topics) by adding a simple theme component.

Comment (tgxworld) [reply assets/stylesheets/common/question-answer.scss:20]: While this is the simplest solution, it is also a solution that is brittle and un-testable. If people are installing both the Q&A plugin and reactions plugin to gether, they will unfortunately have to add a theme component to hide the reactions button if they  don't want it to show. I don't think this plugin should be coupled in anyway to other plugins.

Comment (techAPJ) [reply assets/stylesheets/common/question-answer.scss:20]: I see. What do you suggest here to decouple both the plugins and have reactions disabled by default on Q&A topics?

Comment (tgxworld) [reply assets/stylesheets/common/question-answer.scss:20]: The site owner will have to add their own theme component to hide the reactions if they do not want it. From the Q&A plugin's point of view, the reactions plugin has nothing to do with it.

Comment (techAPJ) [reply assets/stylesheets/common/question-answer.scss:20]: Sounds good. Let's close this PR for now then.

Comment (tgxworld) [reply assets/stylesheets/common/question-answer.scss:20]: @techAPJ If there is a discussion about this on dev, feel free to ping me about it cause I have strong opinions about how plugins should not be coupled with each other.

**Pull request merged**